### ISSUE TITLE
[NEWS] Zasoby /sale/offers/{offerId} - nowa wartość pola “endedBy” / New value in the "endedBy" field for /sale/offers/{offerId} resources

### ISSUE THREAD

**Issue by Lukasz-Zurek** [allegro/allegro-api#4637]
Od **29.06.2021** dla:

- [GET /sale/offers/{offerId}](https://developer.allegro.pl/documentation/#operation/getOfferUsingGET),

- [PUT /sale/offers/{offerId}](https://developer.allegro.pl/documentation/#operation/updateOfferUsingPUT),

zwrócimy nową wartość pola **“endedBy”: EMPTY_STOCK**, która będzie informowała o zakończeniu oferty z powodu wyczerpania liczby sztuk.

**Co się zmieni ?**

- zmienimy powód zakończenia oferty w przypadku jej wyprzedania z **EXPIRATION** na **EMPTY_STOCK**,

- wartość **EMPTY_STOCK** zwrócimy również dla licytacji zakończonej sprzedażą,

- wartość **EXPIRATION** pozostanie dla ofert zakończonych po upływie czasu ich trwania i nie będzie już dotyczyć wyprzedanych ofert.

**Ważne!** Zmiana nie wpłynie na zakończone już oferty.

----

From **June 29th, 2021** for:

- [GET /sale/offers/{offerId}](https://developer.allegro.pl/en/documentation/#operation/getOfferUsingGET),

- [PUT /sale/offers/{offerId}](https://developer.allegro.pl/en/documentation/#operation/updateOfferUsingPUT),

we will return a new value **EMPTY_STOCK** in the **"endedBy"** field, which will inform about the end of the offer due to the stock count exhaustion.

**What will change?**

- we will change the reason for the end of the offer which has been sold out - from **EXPIRATION** to **EMPTY_STOCK**,

- we will also return the **EMPTY_STOCK** value for an auction ended with a sale,

- the **EXPIRATION** value will remain for offers ended after their duration time has expired and will no longer apply to sold-out offers.

**Note!** The change will not affect already ended offers.

**Comment by AnimalDEV** [comment]
Pytanie czy w zdarzeniu `OFFER_ENDED` dla oferty (https://developer.allegro.pl/documentation/#operation/getOfferEvents) też będzie widniał nowy typ w publication.endedBy, bo nie jest zaktualizowana dokumentacja dla tej zmiany?

**Comment by Lukasz-Zurek** [comment]
@AnimalDEV Tak, dla [GET /sale/offer-events](https://developer.allegro.pl/documentation/#operation/getOfferEvents) również będzie zwracana nowa wartość. Proszę spodziewać się zaktualizowanej komunikacji w [naszych aktualnościach](https://developer.allegro.pl/news/).

**Comment by Lukasz-Zurek** [comment]
Dzień dobry / Hello

W treść komunikatu wkradł się błąd - dla licytacji wartość **EMPTY_STOCK** zwrócimy tylko, gdy zakończy się sprzedażą przez Kup Teraz, natomiast dla pozostałych licytacji zwrócimy **EXPIRATION**.

----

There is a mistake in the previous message - we will return the **EMPTY_STOCK** value only for an auction ended with a Buy Now sale, while for the other auctions we will return the **EXPIRATION** value.
